ui/src/components/header/infoPages.ts

```diff
@@ -0,0 +1,15 @@
+// TODO: Replace with BE data
+export const infoPages = [
+  {
+    id: 1,
+    name: 'About',
+    content:
+      '<h1>Heading 1</h1><h2>Heading 2</h2><h3>Heading 3</h3><p>Paragraph</p><p>Paragraph with <a href="https://google.com">link</a></p>',
```

Comment:
@mihow please edit this or let me know when the endpoint is available, so we can show the real content